test-cargo-miri/Cargo.lock

```diff
@@ -1,5 +1,7 @@
 # This file is automatically @generated by Cargo.
 # It is not intended for manual editing.
+version = 3
+
```

Comment:
This looks unrelated... Note that I didn't setup the toolchain using `rustup-toolchain-install-master`, but `rustup override set nightly` instead (because don't want to download another toolchain and don't need to build Miri itself), so this may be inconsistent with what `cargo +miri miri` produces. Maybe I should remove this?

Reply:
Yeah I think for now it'd be better to just remove the lockfile change; I#d like to wait a bit before updating the checked-in lockfiles to the new format.